swiftgen-cli/json.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  SwiftGen
+//  Created by Peter Livesey on 9/16/16.
+//  Copyright (c) 2016 Olivier Halligon
+//  MIT License
+//
+
+import Commander
+import PathKit
+import GenumKit
+
+let jsonCommand = command(
+  outputOption,
+  templateOption(prefix: "json"),
+  templatePathOption,
+  Argument<Path>("FILE", description: "JSON to parse", validator: fileExists)
+) { output, templateName, templatePath, path in
+  let parser = JSONFileParser()
+
+  do {
+    try parser.parseFile(path: path)
```

Comment:
It might be interesting to rewrite your parser to accept multiple input files, see #213 for more information. If you do, don't forget to add unit tests for those.